core/src/loader.rs

```diff
@@ -1743,7 +1741,6 @@ impl<'gc> Loader<'gc> {
                 // We call these methods after we initialize the `LoaderInfo`, but before
                 // we add the loaded clip as a child. The frame constructor should see
                 // 'this.parent == null' and 'this.stage == null'
-                dobj.post_instantiation(uc, None, Instantiator::Movie, false);
```

Comment:
This calls `set_default_instance_name`, which affects AVM2

Reply:
Note that this is also going to be setting flashvars (not actually flashvars in this case, but rather query parameters as flashvars) for AVM1.